src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -2026,6 +2038,15 @@ WorkerExtendedOpNode(MultiExtendedOp *originalOpNode,
 															groupedByDisjointPartitionColumn);
 		workerExtendedOpNode->sortClauseList =
 			WorkerSortClauseList(originalOpNode, groupedByDisjointPartitionColumn);
+		sortClauseList = list_concat(sortClauseList,
```

Comment:
Why do we need `sortClauseList ` at all? can't we operate solely on `workerExtendedOpNode->sortClauseList`?

Reply:
it will come in handy when we need this in window function pushdown (next pr)